Testing/SystemTests/tests/qt/WorkbenchStartupTest.py

```diff
@@ -0,0 +1,79 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import os
+import subprocess
+import sys
+import systemtesting
+
+from mantid.kernel import ConfigService
+from tempfile import NamedTemporaryFile
+
+
+TEST_MESSAGE = "Hello Mantid!"
+TEST_FILE = NamedTemporaryFile(suffix=".txt", delete=False).name.replace('\\', '/')
```

Comment:
Good use of `NamedTemporaryFile`. The only issue with it being create at the module level is that it's possible that the test module is loaded but the test is not run, by using `-R` for example. In this case the temp files would be created but not cleaned up.

Could these be set as instance variables in `__init__`?